code/modules/atmospherics/machinery/portable/pump.dm

```diff
@@ -11,6 +11,12 @@
 	ui_x = 300
 	ui_y = 315
 
+	///Max amount of heat allowed inside of the canister before it starts to melt (different tiers have different limits)
+	var/heat_limit = 5000
+	///Max amount of pressure allowed inside of the canister before it starts to break (different tiers have different limits)
+	var/pressure_limit = 50000
+	max_integrity = 250

```

Comment:
Put redefined vars in the top, and new vars under them for clarity.

Reply:
aight done